estate/models/estate_property.py

```diff
@@ -0,0 +1,28 @@
+
+from odoo import models, fields
+
+class EstateProperty(models.Model):
```

Comment:
```suggestion
from odoo import models, fields


class EstateProperty(models.Model):
```
No need of empty line at the beginning, and then two empty lines are required according to "PEP 8: E302 expected 2 blank lines, found 1"